test/coverlet.core.tests/CoverageTests.cs

```diff
@@ -106,5 +106,35 @@ public void Condition_If()
                 File.Delete(path);
             }
         }
+
+        [Fact]
```

Comment:
To get more mileage out of these tests they can perhaps be theories (at least when only using Debug builds), e.g.:

    [Theory]
    [InlineData(24, 1)]
    [InlineData(26, 2)]
    public void SelectionStatements_Switch(int expectedReturnLineHit, int expectedBranchTaken)

Reply:
If I'm not mistaken with [`Theory`] we run test n times where n is number of `[InlineData]` so in this case could be heavy because every test clone dll and instrument in external process, I mean more load for same result, because inline expectedReturnLineHit,expectedBranchTaken are "strictly related" each other(source code will be the same).
This is the reason I created new custom "asserts" helper.

Reply:
Yes, each InlineData becomes one unit test from the point of running the test. I guess it's difficult to use a test fixture where the test DLL is cloned and instrumented once and then used by all tests and still having it as an external process...

Reply:
>  I guess it's difficult to use a test fixture

Yes...because we could also "instrument" with different parameters(not exposed at the moment https://github.com/tonerdo/coverlet/blob/master/test/coverlet.core.tests/InstrumenterHelper.cs#L168)